modules/notification/tmgmt_notification.module

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Main module file for the TMGMT Notification module.
+ */
+
+use Drupal\tmgmt\JobItemInterface;
+
+/**
+ * Implements hook_theme().
+ */
+function tmgmt_notification_theme() {
+  return [
+    'job_item_needs_review' => [
+      'render element' => 'elements',
+    ],
+    'job_item_autoaccepted' => [
+      'render element' => 'elements',
+    ],
+  ];
+}
+
+/**
+ * Implements hook_tmgmt_courier_trigger_types().
+ */
+function tmgmt_notification_tmgmt_courier_trigger_types(&$types) {
+  $types['job_item_needs_review'] = [
```

Comment:
Indeed i realised also job submission should be an event, since a site owner possibly wants to get notified if someone spends money.